content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -169,130 +169,159 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
+1. Download the [SIG Windows tools](https://github.com/kubernetes-sigs/sig-windows-tools) repository containing install and join scripts
+   ```PowerShell
+   [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+   Start-BitsTransfer https://github.com/kubernetes-sigs/sig-windows-tools/archive/master.zip
+   tar -xvf .\master.zip --strip-components 3 sig-windows-tools-master/kubeadm/v1.15.0/*
+   Remove-Item .\master.zip
+   ```
 
-    If you are behind a proxy, the following PowerShell environment variables must be defined:
+1. Customize the Kubernetes [configuration file](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/v1.15.0/Kubeclustervxlan.json) 
 
-    ```PowerShell
-    [Environment]::SetEnvironmentVariable("HTTP_PROXY", "http://proxy.example.com:80/", [EnvironmentVariableTarget]::Machine)
-    [Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
     ```
+    {
+    "Cri" : {  // Contains values for container runtime and base container setup
+        "Name" : "dockerd", // Container runtime name
+        "Images" : {
+            "Pause" : "mcr.microsoft.com/k8s/core/pause:1.2.0",  // Infrastructure container image
+            "Nanoserver" : "mcr.microsoft.com/windows/nanoserver:1809",  // Base Nanoserver container image
+            "ServerCore" : "mcr.microsoft.com/windows/servercore:ltsc2019"  // Base ServerCore container image
+        }
+    },
+    "Cni" : {  // Contains values for networking executables
+        "Name" : "flannel",  // Name of network fabric
+        "Source" : [{ // Contains array of objects containing values for network daemon(s)
+            "Name" : "flanneld",  // Name of network daemon
+            "Url" : "https://github.com/coreos/flannel/releases/download/v0.11.0/flanneld.exe"  // Direct URL pointing to network daemon executable
+            }
+        ],
+        "Plugin" : {  // Contains values for CNI network plugin
+            "Name": "vxlan" // Backend network mechanism to use: ["vxlan" | "bridge"]
+        },
+        "InterfaceName" : "Ethernet"  // Designated network interface name on Windows node to use as container network
+    },
+    "Kubernetes" : {  // Contains values for Kubernetes node binaries
+        "Source" : {  // Contains values for Kubernetes node binaries 
+            "Release" : "1.15.0",  // Version of Kubernetes node binaries
+            "Url" : "https://dl.k8s.io/v1.15.0/kubernetes-node-windows-amd64.tar.gz"  // Direct URL pointing to Kubernetes node binaries tarball
+        },
+        "ControlPlane" : {  // Contains values associated with Kubernetes control-plane ("Master") node
+            "IpAddress" : "kubemasterIP",  // IP address of control-plane ("Master") node
+            "Username" : "localadmin",  // Username on control-plane ("Master") node with remote SSH access  
+            "KubeadmToken" : "token",  // Kubeadm bootstrap token
```

Comment:
Include that these values can be generated via `kubeadm token create --print-join-command` on the control plane node

Reply:
Included as a note